colaraz/cms/static/js/header.js

```diff
@@ -31,6 +31,13 @@ function headerToggleDropdown() {
     });
 }
 
+function openInviteFriendPopup(inviteFriendUrl) {
+    let windowSize = 'width=802,height=604'
```

Comment:
are we sure we want to have these dimensions constant? What will happen on mobile devices?

Reply:
If we do not give constant dimensions it will get opened in a new tab of chrome which isn't desired. It needs to be a popup window that's why we have to specific constant dimensions identical to colaraz's site.

Reply:
What I meant is that can't we get these dimensions relative to the viewport for a better experience?

Reply:
updated, kindly have a look now.